.rhcicd/clowdapp.yaml

```diff
@@ -79,6 +79,10 @@ objects:
           value: ${SENTRY_DSN}${ENV_NAME}
         - name: QUARKUS_LOG_SENTRY_ENVIRONMENT
           value: ${ENV_NAME}
+        - name: NOTIFICATIONS_ALLOWED_ORG_ID_LIST
+          value: ${NOTIFICATIONS_ALLOWED_ORG_ID_LIST}
+        - name: NOTIFICATIONS_RESTRICT_ACCESS_BY_ORG_ID
+          value: ${NOTIFICATIONS_RESTRICT_ACCESS_BY_ORG_ID}
```

Comment:
I saw that you've set the environment variables like this:

```java
    public static final String ALLOWED_ORG_ID_LIST = "notifications.allowed.orgid.list";
    public static final String RESTRICT_ACCESS_BY_ORG_ID = "notifications.restrict.access.by.orgid";
```

So I guess that we need to change the env vars of the ClowdApp to:

```diff
-NOTIFICATIONS_ALLOWED_ORG_ID_LIST
+NOTIFICATIONS_ALLOWED_ORGID_LIST
```

and 

```diff
-NOTIFICATIONS_RESTRICT_ACCESS_BY_ORG_ID
+NOTIFICATIONS_RESTRICT_ACCESS_BY_ORGID
```
No?